app/src/main/java/org/feature/fox/coffee_counter/data/repository/UserRepositoryInt.kt

```diff
@@ -21,18 +23,24 @@ interface UserRepositoryInt {
 
     suspend fun insertPurchase(purchase: Purchase)
 
+    suspend fun insertImage(image: Image)
+
     suspend fun deleteUser(user: User)
 
     suspend fun deleteFunding(funding: Funding)
 
     suspend fun deletePurchase(purchase: Purchase)
 
+    suspend fun deleteImage(image: Image)
```

Comment:
Pls add Db at the end of the method name to stay consistent and provide better Overview